src/test/java/tests/MilestoneAliasTests.java

```diff
@@ -160,6 +161,97 @@ public void replaceMilestoneAliasesTest() {
+        msFinished1.setDueDate(Optional.of(LocalDate.now().minusDays(1)));
+        msFinished1.setOpen(false);
+
+        TurboMilestone msFinished2 = new TurboMilestone(REPO, 2, "finished2");
+        msFinished2.setDueDate(Optional.of(LocalDate.now().minusDays(2)));
+        msFinished2.setOpen(false);
+
+        TurboMilestone msOngoing0 = new TurboMilestone(REPO, 0, "ongoing0");
+        msOngoing0.setDueDate(Optional.empty());
+        msOngoing0.setOpen(true);
+
+        TurboMilestone msOngoing1 = new TurboMilestone(REPO, 3, "ongoing1");
+        msOngoing1.setDueDate(Optional.empty());
+        msOngoing1.setOpen(true);
+
+        TurboMilestone msOngoing2 = new TurboMilestone(REPO, 4, "ongoing2");
+        msOngoing2.setDueDate(Optional.empty());
+        msOngoing2.setOpen(true);
+
+        model = TestUtils.singletonModel(new Model(REPO,
+               new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(msOngoing0)),
+                new ArrayList<>()
+        ));
+
+        FilterExpression expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current"));
+        List<Qualifier> milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        milestoneQualifiers.stream().forEach(msQ -> {
+            assertEquals("ongoing0", msQ.getContent().get());
+        });
+
+        model = TestUtils.singletonModel(new Model(REPO,
+                new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(msFinished1, msFinished2, msOngoing1)),
+                new ArrayList<>()
+        ));
+
+        expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current"));
+        milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        milestoneQualifiers.stream().forEach(msQ -> {
+            assertEquals("ongoing1", msQ.getContent().get());
+        });
+
+        expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current-2"));
+        milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        milestoneQualifiers.stream().forEach(msQ -> {
+            assertEquals("finished2", msQ.getContent().get());
+        });
+
+        expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current-3"));
+        milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        assertEquals(0, milestoneQualifiers.size());
+
+        model = TestUtils.singletonModel(new Model(REPO,
+                new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(msFinished1, msFinished2, msOngoing1, msOngoing2)),
+                new ArrayList<>()
+        ));
+
+        expr = Qualifier.replaceMilestoneAliases(model, Parser.parse("milestone:current"));
+        milestoneQualifiers = expr.find(Qualifier::isMilestoneQualifier);
+        assertEquals(0, milestoneQualifiers.size());
+    }
```

Comment:
For each test case, you need to explain (using comments) what situation is being tested.